spec/spec_helper.rb

```diff
@@ -7,9 +7,9 @@
   end
 
   # On CI we publish coverage to codecov.io, except on JRuby and TruffleRuby
-  if ENV["CI"] == "true" && RUBY_ENGINE == "ruby"
-    require "codecov"
-    SimpleCov.formatter = SimpleCov::Formatter::Codecov
+  if ENV["CI"] == "true"
```

Comment:
Hi @taichi-ishitani , can you fix the comment above, since now we DO test on all Rubies, but we use the cobertura XML format to allow us to use the more reliable codecov action instead of the unreliable codecov gem?

Reply:
@ms-ati ,
Thank you for reviewing! I've update this comment.